src/lambda/wizards/samInitWizard.ts

```diff
@@ -456,8 +457,11 @@ export class CreateNewSamAppWizard extends MultiStepWizard<CreateNewSamAppWizard
     }
 
     private readonly NAME: WizardStep = async () => {
+        // Default to a name like "lambda-python3.8-1".
+        const defaultName = this.location ? fsutil.getNonexistentFilename(
```

Comment:
`this.location` should never be undefined here. You can remove the ternary and assert `this.location!.path` below.